projects/Galene/default.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  pkgs,
+  sources,
+}@args:
+{
+  metadata = {
+    summary = "Galene is a self-hosted video conferencing server. It features advanced networking and video algorithms and automatic subtitling.";
+    subgrants = [
+      "Galene"
+    ];
+  };
+
+  nixos.modules.programs = {
```

I believe that this should be `services`, since we have [services.galene](https://github.com/NixOS/nixpkgs/blob/nixos-24.11/nixos/modules/services/web-apps/galene.nix) from the [triaged data](https://github.com/ngi-nix/ngipkgs/issues/663)

```suggestion
  nixos.modules.services = {
```